app/models/rent_charge.rb

```diff
@@ -1,4 +1,6 @@
 class RentCharge < ApplicationRecord
+  
+  enum property_type: [:BUN, :FLT, :HOS, :HSE, :MSN]
```

Comment:
Do we know what these mean? Would recommend human-readable enum values over shortened codes. Don't get much value out of the latter, and the former will be more self-documenting.

Assuming bungalow, flat, hostel, house... not sure about MSN? Maisonette?

Reply:
Yes. think so